### ISSUE TITLE
Peer Review- Milestone2 

### ISSUE THREAD

**Issue by Praneta-Paithankar** [airavata-courses/PhotoVault#40]
Hello Team,
 
Good job on developing the user interface. It really looks nice and user-friendly. I also think, Jenkins is configured properly.

However, I am facing few issues:
    1. When I try to search the image, I am not getting the result. It is showing an error. I attached the screenshot for your reference.
   2. Also, if I click on explore public images after searching images I am getting a blank page. 
   3. I am also automatically getting logged out.

Nice to have things:
    1. Also as it is mentioned in previous issue #37, it is nice if you could separate both build process.
    2. Please modify your architecture diagram to show the Consul.
  
Please let me know if you need any further information.

Good luck! 

<img width="1439" alt="exploreissue" src="https://user-images.githubusercontent.com/24793322/53709793-14bb7200-3e08-11e9-82ad-e3053631ffc6.png">
<img width="1438" alt="search" src="https://user-images.githubusercontent.com/24793322/53709794-14bb7200-3e08-11e9-8dcc-0462bfbcbc02.png">
<img width="1440" alt="searchissue" src="https://user-images.githubusercontent.com/24793322/53709795-14bb7200-3e08-11e9-8b78-7d1c43f12c95.png">

**Comment by uteerna** [comment]
Hi Praneeta, 
When I tried to replicate the things you have done I am not facing any issues. We tried it with a new user as well. I am attaching the screenshots for your reference.
<img width="1440" alt="screen shot 2019-03-03 at 11 12 53 pm" src="https://user-images.githubusercontent.com/31369347/53710192-040bfb80-3e0a-11e9-8e0f-b26a1a0d454c.png">
<img width="1440" alt="screen shot 2019-03-03 at 11 13 18 pm" src="https://user-images.githubusercontent.com/31369347/53710193-04a49200-3e0a-11e9-810f-b6e6ea3f2b2f.png">
